packages/adapter-cloudflare-workers/index.js

```diff
@@ -62,15 +62,14 @@ export default function ({ config = 'wrangler.toml' } = {}) {
 			);
 
 			await esbuild.build({
-				platform: 'neutral',
-				mainFields: ['module', 'main'],
-				conditions: ['worker'],
+				platform: 'browser',
```

Comment:
should we add `conditions: ['worker', 'browser']` as they do in their wrangler config (and to Vercel Edge functions as well)?

Reply:
seems reasonable in the cloudflare case. not sure about vercel — someone might be doing something cloudflare-specific? the runtimes aren't 100% identical, might be safer to just play dumb for now